DirectEmployers/MyJobs
# [PD-3351] refactor analytics css

## What's this PR do?
PR refactors and restructures the css for analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics App

## How should this be manually tested?
Pull down repository. Check and make sure inside of analytics that the application still looks the same

## Any background context you want to provide?
